packages/mask/shared/kv-storage.ts

```diff
@@ -39,6 +39,7 @@ export const PersistentStorages = {
             [PluginId.Savings]: false,
             [PluginId.Avatar]: false,
             [PluginId.Trader]: false,
+            [PluginId.Tip]: false,
```

Comment:
Does it necessary for all plugins? If so, please refactor the type to:

`createPersistentKVStorage<Record<PluginId, boolean>>`

Reply:
No, and the type of the key is not `PluginId`.

```
[`${PluginId.ITO}_claim`]: false,
```